src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -73,43 +73,41 @@ const urlIfNoFileValidation = (fieldName: string) =>
 const mustBeMinimum = (fieldName: string, min: number) =>
   `${fieldName} must be at least ${min} characters`;
 
+const mustBeMaximum = (fieldName: string, max: number) =>
+  `${fieldName} must be at least ${max} characters`;
+
+export const roomTitleSchema = Yup.string()
+  .required("Room name is required")
```

Comment:
```suggestion
  .required("Name is required")
```

Reply:
Looks like this wasn't done from https://github.com/sparkletown/sparkle/pull/1341#discussion_r618730811 ?